.travis.yml

```diff
@@ -1,105 +1,86 @@
   global:
     # Doctr deploy key for NSLS-II/NSLS-II.github.io
     - secure: "jUZbT6RuZCGVKEvBCsK22mvXDvGIN10zlHRqj7Q6cl6lf2KaaXnOKb6azCQRJhbvgi9IUkJzjT3ghqeNhjsWOdxzbX4qzTh6jooySP0nyAY+iOQ/F9Q62RdRwM+N5y/t++1Cv2n5BVAYEnGIrE3SyZbeaYLTvNveJnVdeTIKif4="
+    - ETH0_BCAST: "172.17.255.255"
+    - ETH0_IP: "172.17.0.1"
+    - BASE_VER: "R7.0.1.1"
+    - PVA: ""
+    - BUSY_VER: "1-7"
+    - SEQ_VER: "2.2.5"
+    - ASYN_VER: "4-33"
+    - CALC_VER: "3-7"
+    - AUTOSAVE_VER: "5-9"
+    - SSCAN_VER: "2-11-1"
+    - MOTOR_VER: "6-10"
+    - AREADETECTOR_VER: "3-2"
+    - CI_TOP: '${TRAVIS_BUILD_DIR}/.ci'
+    - CI_SCRIPTS: '${TRAVIS_BUILD_DIR}/.ci/ci-scripts'
+    - EPICS_ON_TRAVIS_VERSION: "0.8.1"
+    - EPICS_ON_TRAVIS_PKG: "0.8.1/epics-ci-0.8.1_R7.0.1.1_pva_areadetector3-2_motor6-10.tar.bz2"
+    - EPICS_ON_TRAVIS_URL: 'https://github.com/klauer/epics-on-travis/releases/download/${EPICS_ON_TRAVIS_PKG}'
 
 addons:
   apt:
     packages:
       - graphviz
-
-
-services:
-  - docker
+      - procserv
 
 cache:
   directories:
     - $HOME/.cache/pip
+    - $HOME/.ccache  # https://github.com/travis-ci/travis-ci/issues/5853
 
 matrix:
   include:
     - python: 3.6
-      env: BUILD_DOCS=true CONDA_ENV=1
-    - python: 3.6
-      env: EPICS_DEB_DIST="jessie/staging"
+      env: BUILD_DOCS=true
     - python: nightly
       env: EPICS_DEB_DIST="jessie/staging"
 
 before_install:
-  - export DOCKER0_IP=$(/sbin/ifconfig docker0 |grep 'inet addr' | sed -e 's/.*addr:\([^ ]*\).*/\1/')
-  - export EPICS_CA_ADDR_LIST=$( echo $DOCKER0_IP | sed -e 's/^\([0-9]\+\)\.\([0-9]\+\)\..*$/\1.\2.255.255/' )
-  - export EPICS_CA_AUTO_ADDR_LIST="no"
-  - export EPICS_CA_MAX_ARRAY_BYTES=10000000
-  - export DOCKERIMAGE="klauer/epics-docker"
-  - export EPICS_BASE=/usr/lib/epics
-  - export PE_DOCKERIMAGE="klauer/simioc-docker"
-  - export PE_DOCKERTAG="pyepics-docker"
   - export MPLBACKEND=agg
-  - mkdir /tmp/data
-  - perl --version
   - git fetch --unshallow
-  - docker pull ${DOCKERIMAGE}
-  - docker pull ${PE_DOCKERIMAGE}:${PE_DOCKERTAG}
-  - docker images
-  - docker run -d -p $DOCKER0_IP:7000-9000:5064/tcp -v /tmp/data:/data ${DOCKERIMAGE}
-  - docker run -d -p $DOCKER0_IP:7000-9000:5064/tcp ${PE_DOCKERIMAGE}:${PE_DOCKERTAG}
-  - docker ps -a
-  - ip addr
 
-  - |
-    if [ $CONDA_ENV ]; then
-       wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
-       chmod +x miniconda.sh
-       ./miniconda.sh -b -p /home/travis/mc
-       export PATH=/home/travis/mc/bin:$PATH
-       conda config --set show_channel_urls True
-       conda config --set always_yes True
-    fi
+  # eventually, we should use the specific version tag of epics-on-travis:
+  # - git clone --single-branch --branch "${EPICS_ON_TRAVIS_VERSION}" https://github.com/klauer/epics-on-travis "$CI_TOP"
+  # but for now, get the latest scripts:
+  - git clone --single-branch --branch master https://github.com/klauer/epics-on-travis "$CI_TOP"
+  - sudo ifup docker0
```

Comment:
why bring up the docker network?

Reply:
The docker network addresses seem to be consistent between runs, and I was having trouble with the broadcast traffic of `eth0` for whatever reason.